### PR TITLE
created an installer script

### PR DESCRIPTION
**Pull request by fabibec**
instal.sh compiles the cpp with a name specified by the user and dumps it into /usr/local/bin file. Should work on OSX and Linux. Please review

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ std::string createFileName();
 
 int main(int argc, char* argv[]) {
     if (argc != 2){
-	std::cerr << "Usage: submit filepath" << std::endl;
+	std::cerr << "Usage: " << BASH_NAME << " filepath" << std::endl;
```
Comment (masterYoda8) [inline main.cpp:26]: Where is the Variable/ Constant BASH_NAME specified?

**Review by masterYoda8 — CHANGES_REQUESTED** [review]
Please change filename from instal.sh to install.sh.

File: main.cpp (hunk 1 above)
Comment (fabibec) [reply main.cpp:26]: BASH_NAME is passed via g++ at compile time via the -D option. This ensures that the program uses The name the user chose while installing.
Please look at instal.sh for more details.

**Review by masterYoda8 — CHANGES_REQUESTED** [review]
Please change also name of file in README.md which still contains the old file name.
After changing README.md squash those commits and I will merge.

**Pull request merged**